<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subtreeDialog</class>
 <widget class="QDialog" name="subtreeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose One</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/SR_icon.png</normaloff>:/SR_icon.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="subtreeOK">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>611</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose One :</string>
   </property>
   <widget class="QListWidget" name="subtreeList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>571</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>subtreeOK</sender>
   <signal>accepted()</signal>
   <receiver>subtreeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subtreeOK</sender>
   <signal>rejected()</signal>
   <receiver>subtreeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
